dipy/io/dpy.py

```diff
@@ -39,50 +48,60 @@ def __init__(self, fname, mode='r', compression=0):
         >>> import os
         >>> from tempfile import mkstemp #temp file
         >>> from dipy.io.dpy import Dpy
-        >>> fd,fname = mkstemp()
-        >>> fname = fname + '.dpy' #add correct extension
-        >>> dpw = Dpy(fname,'w')
-        >>> A=np.ones((5,3))
-        >>> B=2*A.copy()
-        >>> C=3*A.copy()
-        >>> dpw.write_track(A)
-        >>> dpw.write_track(B)
-        >>> dpw.write_track(C)
-        >>> dpw.close()
-        >>> dpr = Dpy(fname,'r')
-        >>> A=dpr.read_track()
-        >>> B=dpr.read_track()
-        >>> T=dpr.read_tracksi([0,1,2,0,0,2])
-        >>> dpr.close()
-        >>> os.remove(fname) #delete file from disk
+        >>> def dpy_example():
```

Comment:
The indentation inside the function here seems incorrect. It should be 4 spaces or go back to previous version.